pyglet/image/__init__.py

```diff
@@ -1811,7 +1918,7 @@ def _update_items(self):
                     x += self.item_width + self.column_padding
                 y += self.item_height + self.row_padding
 
-    def __getitem__(self, index):
+    def __getitem__(self, index: int) -> None:
```

Comment:
The return type here is certainly not None. It will be either a single `AbstractImage` or a list of `AbstractImage`.